target/linux/ath79/dts/qca9557.dtsi

```diff
@@ -153,6 +164,51 @@
 				interrupt-map = <0 0 0 0 &pcie0 0>;
 				status = "disabled";
 			};
+
+			wmac: gmac@18100000 {
+				compatible = "qca,qca9550-wmac";
+				reg = <0x18100000 0x230000>;
+
+				interrupt-parent = <&intc2>;
+				interrupts = <1>;
+				status = "disabled";
+			};
+		};
+
+		usb0: usb@1b000000 {
+			compatible = "generic-ehci";
+			reg = <0x1b000000 0x1000>;
```

Comment:
I am not sure, but according to the datasheet, the right size should be: 0x1fc instead of 0x1000. 
I tested 0x1fc and it's working

Reply:
they do work